apps/storybook/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@umami/storybook",
+  "type": "module",
+  "scripts": {
+    "dev": "storybook dev -p 6006",
+    "build": "storybook build -o ./dist"
```

Comment:
right now this package is neither linted nor prettified. you can copy corresponding tasks from our (small-sized) packages